torch/fx/experimental/symbolic_shapes.py

```diff
@@ -5933,22 +5934,20 @@ def simplify(self, expr: _SympyT, size_oblivious: bool = False) -> _SympyT:
         expr = safe_expand(expr)
         expr = self.replace(expr)
 
-        if size_oblivious and expr.has(Max):
-            max_replacements = {}
-            for atom in expr.atoms(Max):
+        if size_oblivious and (expr.has(Max) or expr.has(Min)):  # type: ignore[has-type]
+            min_max_replacements = {}
+            for atom in (*expr.atoms(Max), *expr.atoms(Min)):  # type: ignore[has-type]
```

Comment:
sounds like what we want to do is 
bound a
bound b and return if one of them is def greater or def smaller than the other?